supabase/functions/classify-petiole/index.ts

```
@@ -0,0 +1,771 @@
+    // If we have plan items, create the plan
+    if (planItems && planItems.length > 0 && organizationId) {
+      const planTitle = `AI Draft - ${new Date(record.date).toLocaleDateString('en-IN')} - ${classificationResult.classification.toUpperCase()}`
+
+      const { data: plan, error: planError } = await supabase
+        .from('fertilizer_plans')
+        .insert({
+          farm_id: farmId,
+          organization_id: organizationId,
+          created_by: null, // System generated
+          title: planTitle,
+          notes: `Classification: ${classificationResult.classification}. Reason: ${classificationResult.reason}`,
```

Biome (2.4.11)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)